server/lib/websockets/testing/websockets.ts

```diff
@@ -44,6 +44,9 @@ export class FakeNydusServer
     }
   })
   unsubscribeClient = jest.fn((client: NydusClient, path: string) => {
+    this.fakeSubscriptions.get(path)?.forEach(client => {
```

Comment:
This seems wrong?

The previous implementation of this method removes only a specific client, returning true if the client was subscribed. Your implementation adds to this by telling *every* client that was subscribed to the path that they've been unsubscribed.

I think the more correct implementation here would be to look at the return value below, if true, just do `(client as InspectableNydusClient).unsubscribeClient(client, path))`. There's no real need to go through the roundabout way of retrieving the same value out of the subscriptions

Reply:
You also didn't add similar functionality to `unsubscribeAll` below, which you should definitely do if your intention is to be able to assert against this at the client level